src/lsm/tree_fuzz.zig

```diff
@@ -645,9 +645,14 @@ const Model = struct {
     table_usage: TableUsage,
 
     fn init(table_usage: TableUsage) !Model {
+        const model_node_count = std.math.divCeil(
+            u32,
+            events_max * @sizeOf(Value),
+            NodePool.node_size,
+        ) catch unreachable;
```

Comment:
Nodes don't store values, they store `TableInfo`. Should we be dividing by table's value capacity?

Reply:
These nodes do store values though? That's a different segmented array

Reply:
:facepalm: Ahh I was still thinking about manifestlevels, which isn't what this is for.